plugin-mount/configuration.ui

```diff
@@ -19,25 +19,34 @@
      <property name="title">
       <string>Behaviour</string>
      </property>
-     <layout class="QGridLayout" name="gridLayout">
-      <item row="0" column="0" colspan="2">
+     <layout class="QFormLayout" name="formLayout">
+      <property name="fieldGrowthPolicy">
+       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
+      </property>
+      <property name="labelAlignment">
+       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
+      </property>
+      <item row="0" column="0">
        <widget class="QLabel" name="devAddedLabel">
         <property name="text">
          <string>When a device is connected:</string>
         </property>
        </widget>
       </item>
-      <item row="0" column="1" colspan="2">>
+      <item row="0" column="1">
        <widget class="QComboBox" name="devAddedCombo"/>
       </item>
-      <item row="1" column="0" colspan="2">
+      <item row="1" column="0">
        <widget class="QLabel" name="ejectPressedLabel">
+        <property name="toolTip">
+         <string>Trigger the following action when XF86Eject button is pressed</string>
```

Comment:
How about: "Trigger the following action when XF86Eject button (or other configured shortcut) is pressed" ?